libs/portals/core/src/utils/isCompany.ts

```diff
@@ -0,0 +1 @@
+export { isCompany } from '@island.is/shared/utils'
```

Comment:
Re-exporting can be useful when moving things to keep refactor PRs small. Should avoid otherwise as it can confuse intellisense to suggest bad imports later on (importing portals-core from a backend service).